hooks/post_gen_project.py

```diff
@@ -14,12 +15,21 @@
     "isort",
     "pip",
     "python-dotenv",
+    "setuptools",
+    "wheel",
```

Comment:
@chrisjkuch why do we need setuptools and wheel? These should be unnecessary for a pyproject.toml-based build (this stuff gets specified in pyproject.toml).